lib/msf/core/post/process.rb

```diff
@@ -19,10 +33,35 @@ def has_pid?(pid)
     pid_list.include?(pid)
   end
 
+  #
+  # Gets the `pid` and `name` of the processes on the remote system
+  #
+
```

Comment:
This extra space here isn't necessary. The docs should come right about the method.

Reply:
okay, I'll remove that space :+1: